modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -1,11 +1,182 @@
 [[directives]]
 :description: This page lists all directives available in the Neo4j GraphQL Library.
+
 = Directives
 :page-aliases: directives.adoc
 
 The Neo4j GraphQL Library provides the following directives to be used whilst defining types:
 
-[cols="1,3"]
+== Database mapping
+
+.Database mapping directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship[`@relationship`]
+| Configure  xref::/type-definitions/types/relationships.adoc[relationships] between object types.
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship_properties[`@relationshipProperties`]
+a| Required to help you distinguish between interfaces which are used for relationship properties, and otherwise.
+Can only be used on interfaces, as per its definition:
+[source, graphql, indent=0]
+----
+"""Required to differentiate between interfaces for relationship properties, and otherwise."""
+directive @relationshipProperties on OBJECT
+----
+
+| xref::/type-definitions/directives/database-mapping.adoc#type-definitions-node[`@node`]
+| Specifies the configuration of a GraphQL object type which represents a Neo4j node.
+
+| xref::/type-definitions/directives/database-mapping.adoc#type-definitions-alias[`@alias`]
+| Maps a GraphQL schema field to a Neo4j property on a node or relationship.
+
+| xref::/type-definitions/directives/database-mapping.adoc#_declarerelationship[`@declareRelationship`]
+| Configure xref::/type-definitions/types/relationships.adoc[relationships] to be implemented on object types.
+
+|===
+
+== Security
+
+.Security directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/security/authentication.adoc[`@authentication`]
+| Requires authentication checks when accessing the type.
+
+| xref::/type-definitions/directives/security/authorization.adoc[`@authorization`]
+| Specifies authorization rules for queries and mutations on the type.
+
+| xref::/type-definitions/directives/security/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication filter to include an additional JWT claim.
+
+| xref::/type-definitions/directives/security/configuration.adoc#_nested_claims[`@jwtClaim`]
+| Use in combination with `@jwt`. Configure the JWT authentication filter to include an additional nested JWT claim.
+
+| `@subscriptionsAuthorization`
+| Specifies authorization rules for subscriptions on the type.
+
+|===
+
+== Autogeneration
+
+.Autogeneration directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
+| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+
+| xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-timestamp[`@timestamp`]
+| Flags fields to be used to store timestamps on create/update events.
```

Comment:
Not sure if you are using this formatting for this kind of term. If not, please ignore the suggestions.

```suggestion
| Flags fields to be used to store timestamps on `create` and `update` events.
```